applications/vanilla/views/discussion/helper_functions.php

```diff
@@ -67,14 +67,14 @@ function writeComment($comment, $sender, $session, $currentOffset) {
         static $userPhotoFirst = null;
 
         if ($userPhotoFirst === null) {
-            $userPhotoFirst = c('Vanilla.Comment.UserPhotoFirst', true);
+            $userPhotoFirst = ConfigurationModule::c('Vanilla.Comment.UserPhotoFirst', true);
```

Comment:
I am assuming calling `ConfigurationModule::c` instead of `Gdn::Config` was done for the sake of leveraging the static caching, but this feels like a hack. We're invoking a method from a class that was intended to render a UI component. Is there any way to make this call from a more generic context (i.e. not from a module)?

Reply:
We just discussed with Adam. Are you ok @initvector if I'll move this 'Vanilla.Comment.UserPhotoFirst' to CommentModel ?

Reply:
Sounds good!